### PR TITLE
docs: Highlight the need for Istio configuration to get access logs

### PR DESCRIPTION
**Pull request by juergen-walter**
## Description

Changes proposed in this pull request (what was done and why):

- make documentation more clear

Changes refer to particular issues, PRs or documents:

- SAP internal bug tickets

## Traceability
- [ ] The PR is linked to a GitHub issue.
- [ ] New features have a milestone set.
- [ ] New features have defined acceptance criteria in a corresponding GitHub Issue, and all criteria are satisfied with this PR.
- [ ] The corresponding GitHub issue has a respective `area` and `kind` label.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.
- [ ] Adjusted the documentation if the change is user-facing.
- [ ] The feature is unit-tested
- [ ] The feature is e2e-tested

### CODE REVIEW

File: docs/user/integration/sap-cloud-logging/README.md [hunk 1]
```diff
@@ -103,8 +103,9 @@ The Telemetry module supports the convenient shipment of applications and access
 
 ### Set Up Access Logs
 
-1. By default, Istio access logs are disabled in Kyma. To enable Istio access logs selectively for your workload, follow [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
-   As a result, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
+By default, Istio access logs are disabled in Kyma. After following below procedure, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
+
+1. Enable Istio access logs for your workload following [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
```
Comment (a-thaler) [inline docs/user/integration/sap-cloud-logging/README.md:108]:
```suggestion
1. Enable Istio sidecar injection for your workload following [Enabling Istio Sidecar Injection](https://kyma-project.io/#/istio/user/operation-guides/02-20-enable-sidecar-injection)

1. Enable Istio access logs for your workload following [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
```

**Comment by a-thaler** [comment]
@NHingerl I like to improve it even further to clearly state that Istio needs to be enabled at all for your workload, on top of the suggestion by Juergen. We had issues already were people are not using Istio at all but having Istio access logs enabled, still no data will start flowing. See my two suggestions.

File: docs/user/integration/sap-cloud-logging/README.md [hunk 2]
```diff
@@ -103,8 +103,9 @@ The Telemetry module supports the convenient shipment of applications and access
 
 ### Set Up Access Logs
 
-1. By default, Istio access logs are disabled in Kyma. To enable Istio access logs selectively for your workload, follow [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
-   As a result, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
+By default, Istio access logs are disabled in Kyma. After following below procedure, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:106]:
```suggestion
By default, Istio sidecar injection and Istio access logs are disabled in Kyma. With the following steps, you can analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
```

**Comment by juergen-walter** [comment]
> > @NHingerl I like to improve it even further to clearly state that Istio needs to be enabled at all for your workload, on top of the suggestion by Juergen. We had issues already were people are not using Istio at all but having Istio access logs enabled, still no data will start flowing. See my two suggestions.
> 
> Shall we add that to the `## Prerequisites` section?
> 
> ```
> - The [Telemetry module](../../README.md) is [added](https://kyma-project.io/#/02-get-started/01-quick-install).
> - To ingest Istio access logs, the [Istio module](https://kyma-project.io/#/istio/user/README) must be [added](https://kyma-project.io/#/02-get-started/01-quick-install).
> ```

I'd rather omit in prerequisites because it would duplicate. However, I have no hard opinion, feel free to go with your preferences @a-thaler @NHingerl 

Let's commit suggestions and aim for a quick merge.

**Comment by a-thaler** [comment]
It is not so much about enabling the istio module (which is enabled by default), it is about that you use the sidecar injection and assure that your workload will be part of the mesh (which is not the default unfortunately)

**Comment by NHingerl** [comment]
> It is not so much about enabling the istio module (which is enabled by default), it is about that you use the sidecar injection and assure that your workload will be part of the mesh (which is not the default unfortunately)

Thanks for clarifying! Then we can omit it from the prerequisites (or phrase it as "Make sure the Istio module is added to your cluster."

**Comment by NHingerl** [comment]
However, we should replace the word "enabled" with "added" when talking about modules. Neither in the dashboard, nor in the cli command the word "enable" is used nowadays.

File: docs/user/integration/sap-cloud-logging/README.md [hunk 3]
```diff
@@ -103,8 +103,11 @@ The Telemetry module supports the convenient shipment of applications and access
 
 ### Set Up Access Logs
 
-1. By default, Istio access logs are disabled in Kyma. To enable Istio access logs selectively for your workload, follow [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
-   As a result, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
+By default, Istio sidecar injection and Istio access logs are disabled in Kyma. With the following steps, you can analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
```
Comment (NHingerl) [inline docs/user/integration/sap-cloud-logging/README.md:106]:
```suggestion
The Istio module is added to your Kyma cluster by default, but with Istio sidecar injection and Istio access logs disabled. 

To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```

**Review by a-thaler — DISMISSED** [review]

File: docs/user/integration/sap-cloud-logging/README.md (hunk 3 above)
Comment (a-thaler) [reply docs/user/integration/sap-cloud-logging/README.md:106]: I really think it is not important to talk about the istio module in general but only about if you enabled it for your workload. I somehow like the original better

Comment (NHingerl) [reply docs/user/integration/sap-cloud-logging/README.md:106]: If we make sure that Istio module is mentioned as a prerequisites for ingesting Istio logs etc, then no need to mention it here.

Comment (NHingerl) [reply docs/user/integration/sap-cloud-logging/README.md:106]:
```suggestion
By default, Istio sidecar injection and Istio access logs are disabled in Kyma.

To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```

**Review by NHingerl — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

**Pull request merged**